sql/parser/sql.y

```diff
@@ -1575,8 +1575,7 @@ createdb_opt_item:
 // option will silently break. Best defense is to provide a regression test
 // exercising every such option, at least at the syntax level.
 createdb_opt_name:
-  IDENT {}
-| CONNECTION LIMIT {}
+  CONNECTION LIMIT {}
```

Comment:
I'd rather just get rid of the `createdb_opt_with` and `createdb_opt_list` rules completely. Neither are being used to create nodes in the syntax tree.